app/assets/stylesheets/bootstrap_and_customization.css.scss

```diff
@@ -92,3 +144,26 @@ a:hover {
 .dropdown:hover .dropdown-menu {
   display: block;
 }
+
+.navbar-toggle {
+  margin-right: 25px;
+  margin-top: 22px;
+}
+
+.navbar-collapse {
+  background-color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.